### ISSUE TITLE
Weight calculation based on modelled timesteps only

### ISSUE THREAD

**Issue by waumpel** [tum-ens/urbs#269]
Currently, `weight` is calculated as
```
m.weight = pyomo.Param(
        initialize=float(8760) / (len(m.timesteps) * dt),
        doc='Pre-factor for variable costs and emissions for an annual result')
```
In particular, `timesteps` is used instead of `tm`, which corresponds to `timesteps[1:]`.

I think this is a bug, because `timesteps[0]` is not a part of the modelled time period of an stf. (It is only used for the initial state of storages as far as I know). 

Therefore, `weight` should be calculated as `float(8760) / (len(m.tm) * dt)` instead.

**Comment by ojdo** [comment]
@sonercandas didn't you fix this issue with 7bc9a030cfe5afeb3be88483070b1160d9c0b10a?

**Comment by sonercandas** [comment]
Indeed, this issue can be now closed.
